lib/pf/UnifiedApi/SearchBuilder.pm

```diff
@@ -0,0 +1,248 @@
+package pf::UnifiedApi::SearchBuilder;
+
+=head1 NAME
+
+pf::UnifiedApi::SearchBuilder -
+
+=cut
+
+=head1 DESCRIPTION
+
+pf::UnifiedApi::SearchBuilder
```

Comment:
Perldoc is required for the subroutines in this module

Given the complex nature of this module, I think it would be good to have solid perldoc in it if someone else than @jrouzierinverse has to play in it